api/api-doc/storage_service.json

```diff
@@ -833,6 +833,43 @@
             }
          ]
       },
+      {
+         "path":"/storage_service/repair_status/",
+         "operations":[
+            {
+               "method":"GET",
+               "summary":"Query the repair status and return when the repair is finished or timeout",
+               "type":"string",
+               "enum":[
+                  "RUNNING",
+                  "SUCCESSFUL",
+                  "FAILED"
+               ],
+               "nickname":"repair_async_status_until_finished",
```

Comment:
why "async"? it's a synchronous API.

let's call it repair_await_completion.

Reply:
OK. I like this name.